custom-recipes/pi-system-retrieve-event-frames/recipe.py

```diff
@@ -127,6 +130,10 @@
         row_count = 0
         for row in rows:
             row_count += 1
+            if not use_batch_mode and event_start_time:
+                row["StartTime"] = event_start_time
+            if not use_batch_mode and event_end_time:
+                row["EndTime"] = event_end_time
```

Comment:
Fixed in [c287658](https://github.com/dataiku/dss-plugin-osisoft/pull/35/commits/c287658a84facce059d2e77d62e7be54615d5825)

Reply:
I meant the names of the columns written in the output DS not the names of the variables

Reply:
These two columns are created and named by piwebapi, we are just copying them across the recipe. That's why I think it would be clearer to keep the initial columns names...